docs/rust_wasi_markdown_parser/.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
+
```

Comment:
I think this may no longer be necessary since this is part of the main workspace?